project-ideas/oms/approveTransferOrder.md

```diff
@@ -0,0 +1,44 @@
+# Approve Transfer Orders
+
+1. The process to approve transfer order and correctly update the Order Item Status to either ITEM_PENDING_FULFILLMENT or ITEM_PENDING_RECEIPT on the basis of OH.statusFlowId.
+2. In the core Approve TO service, for each TO, header status update will happen and if the next valid item status transition for its statusFLowId is ITEM_PENDING_FULFILLMENT, the items should be iterated and reservations should happen.
+3. The status update and reservation operations will be done in-line in the Approve service. 
+
```

Comment:
We discussed, adding similar status at OrderHeader level. 
The transition name will be "Approve" and the next status name will be "PENDING_FULFILLMENT". The OrderItems in the TO will be moved to ITEM_PENDING_FULFILLMENT. 

I am taking time to adjust with new status name, Why not APPROVED ?